### PR TITLE
Trivial cleanups2

### PR DESCRIPTION
**Pull request by vdbergh**
Remove glue code at various places.

It should only make a difference for very old tests. So I have not been able to test this.

### CODE REVIEW

**Comment by ppigazzini** [comment]
DEV updated.

**Comment by ppigazzini** [comment]
DEV updated, loade3d a fresh DB (#1234 not ran) I got this:
```
Jan 09 10:51:45 vps1572612706 pserve[25281]: Traceback (most recent call last):
Jan 09 10:51:45 vps1572612706 pserve[25281]:   File "/home/usr00/fishtest/server/env/lib/python3.8/site-packages/waitress/channel.py", line 397, in service
Jan 09 10:51:45 vps1572612706 pserve[25281]:     task.service()
Jan 09 10:51:45 vps1572612706 pserve[25281]:   File "/home/usr00/fishtest/server/env/lib/python3.8/site-packages/waitress/task.py", line 168, in service
Jan 09 10:51:45 vps1572612706 pserve[25281]:     self.execute()
Jan 09 10:51:45 vps1572612706 pserve[25281]:   File "/home/usr00/fishtest/server/env/lib/python3.8/site-packages/waitress/task.py", line 434, in execute
Jan 09 10:51:45 vps1572612706 pserve[25281]:     app_iter = self.channel.server.application(environ, start_response)
Jan 09 10:51:45 vps1572612706 pserve[25281]:   File "/home/usr00/fishtest/server/env/lib/python3.8/site-packages/waitress/proxy_headers.py", line 64, in translate_proxy_headers
Jan 09 10:51:45 vps1572612706 pserve[25281]:     return app(environ, start_response)
Jan 09 10:51:45 vps1572612706 pserve[25281]:   File "/home/usr00/fishtest/server/env/lib/python3.8/site-packages/pyramid/router.py", line 270, in __call__
Jan 09 10:51:45 vps1572612706 pserve[25281]:     response = self.execution_policy(environ, self)
Jan 09 10:51:45 vps1572612706 pserve[25281]:   File "/home/usr00/fishtest/server/env/lib/python3.8/site-packages/pyramid/router.py", line 276, in default_execution_policy
Jan 09 10:51:45 vps1572612706 pserve[25281]:     return router.invoke_request(request)
Jan 09 10:51:45 vps1572612706 pserve[25281]:   File "/home/usr00/fishtest/server/env/lib/python3.8/site-packages/pyramid/router.py", line 245, in invoke_request
Jan 09 10:51:45 vps1572612706 pserve[25281]:     response = handle_request(request)
Jan 09 10:51:45 vps1572612706 pserve[25281]:   File "/home/usr00/fishtest/server/env/lib/python3.8/site-packages/pyramid/tweens.py", line 43, in excview_tween
Jan 09 10:51:45 vps1572612706 pserve[25281]:     response = _error_handler(request, exc)
Jan 09 10:51:45 vps1572612706 pserve[25281]:   File "/home/usr00/fishtest/server/env/lib/python3.8/site-packages/pyramid/tweens.py", line 17, in _error_handler
Jan 09 10:51:45 vps1572612706 pserve[25281]:     reraise(*exc_info)
Jan 09 10:51:45 vps1572612706 pserve[25281]:   File "/home/usr00/fishtest/server/env/lib/python3.8/site-packages/pyramid/util.py", line 733, in reraise
Jan 09 10:51:45 vps1572612706 pserve[25281]:     raise value
Jan 09 10:51:45 vps1572612706 pserve[25281]:   File "/home/usr00/fishtest/server/env/lib/python3.8/site-packages/pyramid/tweens.py", line 41, in excview_tween
Jan 09 10:51:45 vps1572612706 pserve[25281]:     response = handler(request)
Jan 09 10:51:45 vps1572612706 pserve[25281]:   File "/home/usr00/fishtest/server/env/lib/python3.8/site-packages/pyramid/router.py", line 143, in handle_request
Jan 09 10:51:45 vps1572612706 pserve[25281]:     response = _call_view(
Jan 09 10:51:45 vps1572612706 pserve[25281]:   File "/home/usr00/fishtest/server/env/lib/python3.8/site-packages/pyramid/view.py", line 674, in _call_view
Jan 09 10:51:45 vps1572612706 pserve[25281]:     response = view_callable(context, request)
Jan 09 10:51:45 vps1572612706 pserve[25281]:   File "/home/usr00/fishtest/server/env/lib/python3.8/site-packages/pyramid/viewderivers.py", line 427, in rendered_view
Jan 09 10:51:45 vps1572612706 pserve[25281]:     result = view(context, request)
Jan 09 10:51:45 vps1572612706 pserve[25281]:   File "/home/usr00/fishtest/server/env/lib/python3.8/site-packages/pyramid/viewderivers.py", line 113, in _class_requestonly_view
Jan 09 10:51:45 vps1572612706 pserve[25281]:     response = getattr(inst, attr)()
Jan 09 10:51:45 vps1572612706 pserve[25281]:   File "/home/usr00/fishtest/server/fishtest/api.py", line 378, in update_task
Jan 09 10:51:45 vps1572612706 pserve[25281]:     result = self.request.rundb.update_task(
Jan 09 10:51:45 vps1572612706 pserve[25281]:   File "/home/usr00/fishtest/server/fishtest/rundb.py", line 871, in update_task
Jan 09 10:51:45 vps1572612706 pserve[25281]:     return self.sync_update_task(worker_info, run_id, task_id, stats, spsa)
Jan 09 10:51:45 vps1572612706 pserve[25281]:   File "/home/usr00/fishtest/server/fishtest/rundb.py", line 961, in sync_update_task
Jan 09 10:51:45 vps1572612706 pserve[25281]:     fishtest.stats.stat_util.update_SPRT(updated_results, sprt)
Jan 09 10:51:45 vps1572612706 pserve[25281]:   File "/home/usr00/fishtest/server/fishtest/stats/stat_util.py", line 232, in update_SPRT
Jan 09 10:51:45 vps1572612706 pserve[25281]:     elo_model = sprt.get["elo_model"]
Jan 09 10:51:45 vps1572612706 pserve[25281]: TypeError: 'builtin_function_or_method' object is not subscriptable

```

**Comment by vdbergh** [comment]
Hmmm.... `elo_model = sprt.get["elo_model"]` Stupid typo....

File: server/fishtest/stats/stat_util.py [hunk 1]
```diff
@@ -251,7 +246,7 @@ def update_SPRT(R, sprt):
 
     R_ = R.get("pentanomial", R3)
 
-    batch_size = sprt.get("batch_size", 1)
+    batch_size = sprt.get["batch_size"]
```
Comment (ppigazzini) [inline server/fishtest/stats/stat_util.py:249]: I-m wondering if this should be:
`batch_size = sprt["batch_size"]`

```
Jan 09 11:14:53 vps1572612706 pserve[29311]:   File "/home/usr00/fishtest/server/fishtest/api.py", line 378, in update_task
Jan 09 11:14:53 vps1572612706 pserve[29311]:     result = self.request.rundb.update_task(
Jan 09 11:14:53 vps1572612706 pserve[29311]:   File "/home/usr00/fishtest/server/fishtest/rundb.py", line 871, in update_task
Jan 09 11:14:53 vps1572612706 pserve[29311]:     return self.sync_update_task(worker_info, run_id, task_id, stats, spsa)
Jan 09 11:14:53 vps1572612706 pserve[29311]:   File "/home/usr00/fishtest/server/fishtest/rundb.py", line 961, in sync_update_task
Jan 09 11:14:53 vps1572612706 pserve[29311]:     fishtest.stats.stat_util.update_SPRT(updated_results, sprt)
Jan 09 11:14:53 vps1572612706 pserve[29311]:   File "/home/usr00/fishtest/server/fishtest/stats/stat_util.py", line 249, in update_SPRT
Jan 09 11:14:53 vps1572612706 pserve[29311]:     batch_size = sprt.get["batch_size"]

```

Comment (vdbergh) [reply server/fishtest/stats/stat_util.py]: Yep.... Dyslexia.

**Review by ppigazzini — APPROVED** [review]
Looks good to me :)
No entries in logs browsing new and old tests, clicking on user name or worker name, starting `delta_update_users.py` and adding/stopping runs.

**Comment by ppigazzini** [comment]
Please squash when completed :)

**Comment by vdbergh** [comment]
I squashed the PR but I discovered a cosmetic bug at last moment. I mistranslated a line. Below is the corrected version:
```
            % if approver and 'worker_info' in task and 'username' in task['worker_info'] and task['worker_info']['username'] != "Unknown_worker":
+            % if approver and task['worker_info']['username'] != "Unknown_worker":
```
Can you check if you log in as approver the test page is ok?

**Comment by ppigazzini** [comment]
Ok, I will update DEV and test the feature.
I triple tested that change because was a bit strange (url shown also for no approvers) anyway it worked for not logged user, logged user, approver.

**Comment by ppigazzini** [comment]
Yes

**Comment by ppigazzini** [comment]
PROD updated, thank you @vdbergh :)

**Comment by vondele** [comment]
was there an issue, the events log is full of failed updates?

**Comment by ppigazzini** [comment]
@vdbergh the squash has lost the fixes for `batch_size = sprt.get["batch_size"]` and `elo_model = sprt.get["elo_model"]`
I fixed PROD by hand, I'm opening a PR.

**Pull request merged**